articles/multifactor-authentication/administrator/sms-templates.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Customize SMS Messages
+---
+
+# Customize SMS Messages
+
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
```

Comment:
To customize the content of the messages sent by Auth0 during enrollment (when associating a device to Guardian) or verification (when an authentication messages is sent to the device), do the following.